man/chpl.txt

```diff
@@ -587,9 +587,6 @@ ENVIRONMENT VARIABLES FOR OPTIONS
   non-empty value of the environment variable equivalent has the same effect
   as passing that option once.
 
-
-ENVIRONMENT
-
   See $CHPL_HOME/doc/chplenv.rst for detailed information about
   general environment variable settings, legal values, and default
   settings. Run $CHPL_HOME/util/printchplenv to view your current
```

Comment:
I think `CHPL_MODULE_PATH` is important enough to stay there for now. `CHPL_DEVELOPER` has a `--devel flag`, but it is not explicitly documented in the man page. Should we add this somewhere?